src/main.cpp

```diff
@@ -23,11 +23,11 @@ int main(int argc, char** argv)
 	// Create characters
 	//TODO Add weapons
 	std::string nameone = "Marcus";
-	std::string nametwo = "Lalala";
-	FCharacter one(std::vector<FWeapon>(), nameone, 1, false);
-	FCharacter two(std::vector<FWeapon>(), nametwo, 2, true);
+	//std::string nametwo = "Lalala";
+	FCharacter one(FVector(50.f, 150.f), FVector(20.f, 40.f), std::vector<FWeapon>(), nameone, 1, false);
+	//FCharacter two(std::vector<FWeapon>(), nametwo, 2, true);
 	engine->addPlayer(one);
```

Comment:
This method should take a string argument which would be the filename to the texture of this character.